crates/sui-types/src/storage.rs

```diff
@@ -524,11 +524,36 @@ impl InMemoryStore {
             .insert(sequence_number, digest);
     }
 
+    pub fn delete_checkpoint_content(&mut self, sequence_number: u64) -> anyhow::Result<()> {
```

Comment:
it looks like this is only being used from tests - can you add `_for_tests` to the method name?

Reply:
for sure, will do.

Reply:
done